agent/agent-tooling/build.gradle.kts

```diff
@@ -76,7 +77,6 @@ dependencies {
   testImplementation("io.opentelemetry:opentelemetry-sdk-metrics")
   testImplementation("io.opentelemetry:opentelemetry-sdk-logs")
   testImplementation("io.opentelemetry:opentelemetry-sdk-testing")
-  testImplementation("io.opentelemetry:opentelemetry-sdk-metrics-testing")
```

Comment:
this is needed for InMemoryMetricExporter. i don't see that change if you remove this.

Reply:
When Metrics SDK went stable in 1.14.0, `InMemoryMetricExporter` was moved to the (stable) opentelemetry-sdk-testing